buildSrc/src/main/kotlin/Dependencies.kt

```diff
@@ -12,7 +12,7 @@ object Versions {
     const val javaTarget = "1.8"
 
     const val kotlin = "1.2.61"
-    const val gradleDocker = "3.6.0"
+    const val gradleDocker = "3.2.4"
```

Comment:
Ouch. Why?

Reply:
It fails with
```
> Task :deployment:docker:docker FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':deployment:docker:docker'.
> Could not resolve all files for configuration ':deployment:docker:dockerJava'.
   > Could not find com.aries:docker-java-shaded:3.1.0-rc-3.
     Searched in the following locations:
       - file:/Users/destab/.m2/repository/com/aries/docker-java-shaded/3.1.0-rc-3/docker-java-shaded-3.1.0-rc-3.pom
       - file:/Users/destab/.m2/repository/com/aries/docker-java-shaded/3.1.0-rc-3/docker-java-shaded-3.1.0-rc-3-cglib.jar
       - https://repo.maven.apache.org/maven2/com/aries/docker-java-shaded/3.1.0-rc-3/docker-java-shaded-3.1.0-rc-3.pom
       - https://repo.maven.apache.org/maven2/com/aries/docker-java-shaded/3.1.0-rc-3/docker-java-shaded-3.1.0-rc-3-cglib.jar
       - http://artifacts.codice.org/content/repositories/thirdparty/com/aries/docker-java-shaded/3.1.0-rc-3/docker-java-shaded-3.1.0-rc-3.pom
       - http://artifacts.codice.org/content/repositories/thirdparty/com/aries/docker-java-shaded/3.1.0-rc-3/docker-java-shaded-3.1.0-rc-3-cglib.jar
     Required by:
         project :deployment:docker
```
I see other people are having that issue too https://github.com/bmuschko/gradle-docker-plugin/issues/467